AutocompleteClient/FW/Logic/Request/Builder/RequestBuilder+QueryItems.swift

```diff
@@ -222,4 +222,9 @@ extension RequestBuilder {
         guard let versionId = versionId else { return }
         queryItems.add(URLQueryItem(name: Constants.Quiz.versionId, value: versionId))
     }
+    
```

Comment:
Let's change this to be `quizVersionId` (couple lines above) and `quizSessionId`